pennylane_lightning/core/src/utils/RuntimeInfo.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+// Copyright 2018-2023 Xanadu Quantum Technologies Inc. and contributors.
```

Comment:
Xanadu is generally the copyright holder for code in the PL ecosystem:
```suggestion
// Copyright 2018-2023 Xanadu Quantum Technologies Inc.
```